examples/dockervolume/helpers.sh

```diff
@@ -0,0 +1,32 @@
+#!/usr/bin/env bash
+set -euo pipefail
+
+create_docker_volume() {
+  # Create Docker volume if it doesn't already exist
+  docker volume ls | grep myvol || \
+    docker volume create myvol
+
+  # Add files to volume via a helper container
+  docker run -v myvol:/data --name helper busybox true
```

Comment:
I think this can be simplified even further to just use create, then copy then rm. You don't actually have to run the container.

https://stackoverflow.com/a/55683656

Reply:
Nice, thank you!  Updated.